public/components/common/search/sql_search.tsx

```diff
@@ -95,6 +95,7 @@ export const DirectSearch = (props: any) => {
   } = props;
 
   const explorerSearchMetadata = useSelector(selectSearchMetaData)[tabId];
+  const explorerSearchMeta = useSelector(selectSearchMetaData)[tabId];
```

Comment:
the one above is the search metadata, why need to add a additional one?

Reply:
didn't look at what explorerSearchMeta was before reusing for this file, removed that line